### PR TITLE
Try makefile from concerto

### PR DESCRIPTION
**Pull request by richardkapolnai-da**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -33,6 +33,18 @@ commands:
           fail_only: true
           only_for_branches: main
           webhook: '${SLACK_WEBHOOK}'
+  install_yq:
```
Comment (tamaskalcza-da) [inline .circleci/config.yml:36]: Minor: I would include this into the `install_sdk` command and rename it to something like `install_prerequisites`.

File: .circleci/config.yml [hunk 2]
```diff
@@ -33,6 +33,18 @@ commands:
...
+    steps:
+      - run:
+          name: Install pip3
+          command: |
+            sudo apt-get update || sudo apt-get update
+            sudo apt-get install python3-pip
+      - run:
+          name: Install yq with pip3
+          command: |
+            pip3 install yq
```
Comment (tamaskalcza-da) [inline .circleci/config.yml:47]: Is this enough? I mean as far as I know `yq` depends on `jq`. Don't you have install that as well? 🤔

File: README.md [hunk 3]
```diff
@@ -35,7 +35,8 @@ The demo shows how Daml supports core features of CBDC, addressing these key req
 - [Daml SDK](https://docs.daml.com/)
 - Make
 - Python 3
-- Node v12
+- Node v14
+- Yq (hint: `pip3 install yq`)
```
Comment (tamaskalcza-da) [inline README.md:39]: Minor:
```suggestion
- [yq](https://pypi.org/project/yq/)
```

File: daml.mk [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+ui/daml.js: $(jsdars)
+	$(daml) codegen js $^ -o $@
+	@touch $@
+
+.PHONY: $(phony-test-targets)
+$(phony-test-targets): build-dars
+	$(daml) test --project-root $(project-root) --junit $(PWD)/target/$(@F).xml
+
+.SECONDEXPANSION:
```
Comment (richardkapolnai-da) [inline daml.mk:32]: (this is the magic, idea borrowed from the original example. pls see make manual for this if you are interested.)

Comment (richardkapolnai-da) [reply daml.mk:32]: we can replace it later, automatically creating the dependency file by make, as in the current version

File: .circleci/config.yml (hunk 2 above)
Comment (richardkapolnai-da) [reply .circleci/config.yml:47]: Good catch! However, in the CI, it's enough (it was green).

File: .circleci/config.yml (hunk 1 above)
Comment (richardkapolnai-da) [reply .circleci/config.yml:36]: I introduced instead, what do you think?

Comment (tamaskalcza-da) [reply .circleci/config.yml:36]: Don't have a strong opinion, I'm fine either way. The two approaches are not mutually exclusive. You can define this and create `install_prerequisites` where you use it. I'll leave it up to you.

**Review by tamaskalcza-da — APPROVED** [review]

**Pull request merged**